addons/web/static/src/scss/bootstrap_review_frontend.scss

```diff
@@ -136,11 +136,26 @@ $o-navbar-nav-pills-link-border-radius: $nav-pills-border-radius !default;
     left: 50%; // Avoid horizontal scrollbar in Chrome
 }
 
+// Pagination
 .pagination {
-    // Ensure circle for one digit numbers
     font-variant-numeric: tabular-nums;
 }
 
+// Ensure circle pagination
+.page-link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    line-height: 0;
+    width: 2.5em;
+    aspect-ratio: 1 / 1;
+}
+
+// Ensures the readability of numbers when there are many (>3) and they overflow the circle.
```

Comment:
- Max 80 chars per line for comments
- `Ensure` instead of `Ensures` (like the one you wrote above)